pages/k8s/upgrading.md

```diff
@@ -95,35 +95,19 @@ By default, **Charmed Kubernetes** 1.15 and later use Containerd as the containe
 runtime. This subordinate charm can be upgraded with the command:
 
 ```bash
-juju upgrade-charm containerd
+juju upgrade-charm containerd --channel=1.24/stable
```

Comment:
this only works with juju 2.9, it doesn't work with older juju controller b/c they cannot reach get tracked channels from charmhub.  I imagine that's why it wasn't updated but its worth a discussion